apps/wallet/src/ui/app/pages/home/nft-transfer/useTransferKioskItem.ts

```diff
@@ -33,6 +33,12 @@ export function useTransferKioskItem({
 	const obPackageId = useFeatureValue('kiosk-originbyte-packageid', ORIGINBYTE_PACKAGE_ID);
 	const { data: kioskData } = useGetKioskContents(address);
 
+	const kioskClient = new KioskClient({
```

Comment:
This would need to be memo'd or something similar, or actually probably just constructed dynamically alongside the transactionblock instead of here.